plugin/healthchecker/README.md

```diff
@@ -28,9 +28,9 @@ HTTP method can be configured in the following format: `http OR http:PORT OR htt
 
 - `CACHE_SIZE` -- maximum number of records in cache
 - `HEALTHCHECK_INTERVAL` -- time interval of updating status of records in cache in duration format
-- `ORIGIN.` -- DNS origin. Must end with a dot.
```

Comment:
Maybe we should keep an option to specify origin and use regexp as alternative way. 
/cc @realloc

Reply:
Yes. As we discussed earlier, the `ORIGIN` or `@` alias must be supported, but it has to be optional.